packages/core/test/integration/query-interface/bulk-insert.test.ts

```
@@ -0,0 +1,52 @@
+  });
+
+  beforeEach(async function createTestTable() {
+    await this.queryInterface.createTable('UsersBulkInsert', {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+      name: DataTypes.STRING,
+    });
+  });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)